RelayMetadata+CoreDataClass.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  RelayMetadata+CoreDataClass.swift
+//  Nos
+//
+//  Created by Martin Dutra on 8/6/23.
+//
+//
+
+import Foundation
+import CoreData
+
+public class RelayMetadata: NosManagedObject {
```

Comment:
Given that this model has a 1-to-1 relationship with `Relay`, should we just move all the properties on `RelayMetadata` to `Relay`? It will make them easier and cheaper to access.

Reply:
Done